### PR TITLE
adding style to FollowArtistPopover

### PR DESCRIPTION
**Pull request by oxaudo**
For now I'm removing the react-popover addition and just putting this pop up statically below the follow element.

We can figure out later - what artsy popover elements should look like.
For now also using the existing size for Avatar. May need to introduce a smaller `xxs` size to palette if @design decides that it is crutial.

For now the popover looks like that:
screen shot 2018-12-10 at 3 28 14 pm

### CODE REVIEW

**Comment by artsyit** [comment]
Deploy preview for *artsy-reaction* ready!

Built with commit 3985b119ef4019dd96fe6c73b58ffe7669cadf80

https://deploy-preview-1662--artsy-reaction.netlify.com

File: package.json [hunk 1]
```diff
@@ -175,7 +175,6 @@
     "react-markdown": "^2.5.0",
     "react-oembed-container": "^0.3.0",
     "react-overlays": "^0.8.3",
-    "react-popover": "^0.5.10",
```
Comment (damassi) [inline package.json]: 👍

File: src/Components/FollowButton/FollowArtistButton.tsx [hunk 2]
```diff
@@ -52,16 +49,19 @@ interface State {
   openSuggestions: boolean
 }
 
+const SuggestionsPopoverContainer = styled(Box)`
+  position: absolute;
+  z-index: 1000;
```
Comment (damassi) [inline src/Components/FollowButton/FollowArtistButton.tsx]: Mind experimenting with the lowest possible index?

**Comment by damassi** [comment]
Looks great 👍

File: src/Components/FollowButton/FollowArtistButton.tsx (hunk 2 above)
Comment (oxaudo) [reply src/Components/FollowButton/FollowArtistButton.tsx]: Hah... Sure. I'm not even sure why the contact button was above this pop up. So I went with the random really high number. Will definitely try to make it a more reasonable number.

**Pull request merged**